virttest/qemu_devices/qdevice_format.py

```diff
@@ -306,9 +306,12 @@ def _device_json_format(self, params):
         """
         dev_type = "device"
         driver = params.get("driver")
-        driver = "usb_driver" if driver.startswith("usb-") else driver
         if driver not in self._device_driver_checked:
             self._device_driver_checked.append(driver)
+            if driver.startswith("usb-"):
+                self._special_args_in_json[dev_type][driver] = (
+                    self._special_args_in_json[dev_type]["usb_driver"]
+                )
```

Comment:
Hi @nickzhq 
As the related issue is that it does not support converting the `bootindex` to `int`  data type, so I assume that we just add `"bootindex": self._str_to_dec` to the `usb_driver` part to support it.
https://github.com/avocado-framework/avocado-vt/blob/c5bd80265dd43d6e18b905a9eb2aced3b162a954/virttest/qemu_devices/qdevice_format.py#L92-L97
But as the commit message `- Optimizing and Fixing the error about usb args type for each usb-devices.` and your code, would you like to provide more about the optimization and errors? Thanks.